src/test/java/org/opentripplanner/routing/stoptimes/StopTimesHelperTest.java

```diff
@@ -41,6 +41,25 @@ public static void setUp() throws Exception {
     tt.getTripTimes(0).cancelTrip();
   }
 
+  /**
+   * Case 0, requested number of departure = 0
+   */
+  @Test
+  void stopTimesForStop_zeroRequestedNumberOfDeparture() {
+    // Case 1, should find first departure for each pattern
```

Comment:
Comment is incorrect (copied from test below).

Reply:
Removed both comments since they are redundant with the javadoc

Reply:
Great solution to the problem! :stuck_out_tongue: